content/en/docs/installation.md

```diff
@@ -1,675 +0,0 @@
----
```

Comment:
no because we can't determine what they're going to installation for
it could be any of
- installation
- upgrade
- uninstall
- bootstrap

We should update the 404 page to list pages they could have been looking for
example

```

Were you looking for:
- Documentation
- Blog
- Roadmap
- Support

```

Reply:
It's `installation.md`, so redirecting to installation should be good...?

Reply:
but installation.md could mean any of 

- installation
- upgrade
- uninstall
- bootstrap

because thats all the content in the installation.md iirc